src/base/print_trace.C

```diff
@@ -204,8 +204,9 @@ void print_trace(std::ostream & out_stream)
 
   // Let the user disable GDB backtraces by configuring with
   // --without-gdb-command or with a command line option.
-  if (std::string(LIBMESH_GDB_COMMAND) != std::string("no") &&
-      !libMesh::on_command_line("--no-gdb-backtrace"))
+  if ((std::string(LIBMESH_GDB_COMMAND) != std::string("no") &&
+       !libMesh::on_command_line("--no-gdb-backtrace")) ||
+      libMesh::command_line_value("gdb", 0))
```

Comment:
Hmm... from the docs for `libMesh::command_line_value`
```
/**
 * \returns The value associated with name on the command line if it is specified,
 * otherwise return the default, provided value.  A second template function is provided
 * to support recognizing multiple variations of a given option
 */
```
So I guess what you are saying here is that if someone passes `--gdb` (with no trailing arg) on the command line, then `libMesh::command_line_value` would return a non-zero value? I'm not sure about this... any reason to not just use `libMesh::on_command_line("--gdb")` instead?

Reply:
I just didnt know about that one. 
It's changed now